src/main/resources/liquibase/crop_changelog/16_1_0.xml

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="medramos" id="v16.1.0-1">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="germplsm" columnName="germplsm_uuid"/>
+			</not>
+		</preConditions>
+		<comment>
+			Add guid column to germplsm table
+		</comment>
+		<addColumn tableName="germplsm">
+			<column name="germplsm_uuid" type="VARCHAR(36)">
+				<constraints nullable="true"/>
+			</column>
+		</addColumn>
+	</changeSet>
+
+	<changeSet author="mderamos" id="v16.1.0-2">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT
+				COUNT(*)
+				FROM
+				information_schema.routines
+				WHERE
+				routine_schema = DATABASE()
+				AND specific_name = 'populate_germplsm_uuid_values';
+			</sqlCheck>
+		</preConditions>
+		<sql dbms="mysql" splitStatements="false">
+			DROP PROCEDURE populate_germplsm_uuid_values;
+		</sql>
+	</changeSet>
+
+	<changeSet author="mderamos" id="v16.1.0-3">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT
+				COUNT(*)
+				FROM
+				information_schema.routines
+				WHERE
+				routine_schema = DATABASE()
+				AND specific_name = 'populate_germplsm_uuid_values';
+			</sqlCheck>
+		</preConditions>
+		<comment>
+			Create Stored procedure that populate the germplsm_uuid values
+		</comment>
+		<sql dbms="mysql" splitStatements="false">
+			CREATE PROCEDURE populate_germplsm_uuid_values()
+			BEGIN
+
+			DECLARE useUuid INT DEFAULT 1;
+			DECLARE hasNullUuidCount INT DEFAULT 1;
+			DECLARE duplicates INT DEFAULT 1;
+
+			SELECT use_uuid
+			INTO useUuid
+			FROM workbench.workbench_crop
+			WHERE db_name = DATABASE();
+
+			IF useUuid = 0	THEN
+
+			WHILE(hasNullUuidCount > 0) DO
+
+			UPDATE germplsm g
+			SET germplsm_uuid = CONCAT((SELECT plot_code_prefix
+			FROM workbench.workbench_crop wc
+			WHERE db_name = DATABASE()),
+			'G',
+			RIGHT(sha1(CONCAT(uuid(), g.gid)), 15))
+			WHERE g.germplsm_uuid IS NULL;
```

Comment:
Is there other way to uniquely generate non uuid value?

Reply:
I don't think there is. It's also what the BA team suggested. The ticket was tested in both centos-2 and 4 servers. It took 23:11minutes to complete the execution of liquibase scripts for use_uuid=0 for ALL crops.

Reply:
To give more context on how this approach started: The [v13.3.0-4 changeset](https://github.com/IntegratedBreedingPlatform/Middleware/blob/e61b9aae8719384033e1655eb397642bd41fe1e9/src/main/resources/liquibase/crop_changelog/13_3_0.xml#L61), is in turn a simplified version of the original plot_id (obs_unit_id) migration: 
https://github.com/IntegratedBreedingPlatform/Middleware/commit/70b4eb14111602f5fd511cc4327be35a51038db8, an approach that also checks for duplicates in several iterations.

The first thing that I tried with plot_id was mysql `rand()`. I tested it on cimmyt db with an nd_experiment table of ~ 22 million rows, but I saw that the approach generated a lot of duplicates. The update was something like this:
```sql
DECLARE chars CHAR(62)  DEFAULT 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789';

UPDATE test_table
SET plot_id = CONCAT((SELECT plot_code_prefix
  FROM
	workbench.workbench_crop wc
  WHERE
	db_name = DATABASE()),
 'P', concat(
	 substring(chars, rand() * 62 + 1, 1), 
	 substring(chars, rand() * 62 + 1, 1),
	 substring(chars, rand() * 62 + 1, 1),
	 substring(chars, rand() * 62 + 1, 1),
	 substring(chars, rand() * 62 + 1, 1),
	 substring(chars, rand() * 62 + 1, 1)))
WHERE plot_id IS NULL;
```
After each iteration I had to remove duplicates, but I couldn't measure how long it'll take, I stopped testing after several hours.

Then I decided to try something involving `uuid()`, which is the approach wee see in https://github.com/IntegratedBreedingPlatform/Middleware/commit/70b4eb14111602f5fd511cc4327be35a51038db8. 
The `sha1()` part was so that the final result look more like a random string, given that [myql's uuid v1](https://dev.mysql.com/doc/refman/5.6/en/miscellaneous-functions.html#function_uuid) is a timestamp-based uuid.
I don't know why, but on `nd_experiment`  with 22M rows, even this alternative generated some duplicates. so I had to run the statement several times, checking for duplicates after each iteration. 

Luckily in the case of `ims_lot` and `germplsm` we don't see these "duplicates", and so the script is more simple and the update can be done in one pass.

cc @mderamos @abatac @darla-leafnode @clarysabel